lib/prototype-admin/clears-data.html

```diff
@@ -0,0 +1,34 @@
+
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Clear data? | GOV.UK Prototype Kit
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ govukBackLink({
+          "text": "Back",
+          "href": "javascript: window.history.go(-1)"
+        }) 
+      }}
+
+      <h1 class="govuk-heading-xl">
+        Are you sure?
```

Comment:
It would be good to get the heading and the title to be the same here if possible.